isaric/parsers/idor-brazil.toml

```diff
@@ -267,6 +267,10 @@
   field = "healthwork_erterm"
   ref = "Y/N/NK"
 
+[subject.works_lab]
+  field = "labwork_erterm"
+  ref = "Y/N/NK"
+
```

Comment:
same comment for mapping only Y to works_lab where the question asks specifically about microbiology lab work